test-apps/display-performance-test-app/vite.config.ts

```diff
@@ -60,21 +66,24 @@ export default defineConfig(() => {
           /node_modules/, // prevent errors for modules
           /core\/frontend/, // prevent errors with require in IModelApp
         ],
-        transformMixedEsModules: true // transforms require statements
+        transformMixedEsModules: true, // transforms require statements
       },
       rollupOptions: {
-        input: path.resolve(__dirname,"index.html"),
+        input: path.resolve(__dirname, "index.html"),
         // run `rushx build --stats` to view stats
+        logLevel: process.env.CI ? "silent" : "warn",
```

Comment:
Checking the logs in azure pipelines I noticed the warnings weren't muted for DPTA - the `process.env.CI` also returned undefined. Does azure dev ops not set CI to false automatically? Should we change our rush build command in the yaml file to have CI set to true?

Reply:
Added a `VITE_CI` flag to address only packages that uses vite for building.